styles.css

```diff
@@ -1,6 +1,108 @@
-body{
-    color: #bae1ff;
+:root {
+    --primary-color: #370140;
+    --secondary-color: #bae1ff;
 }
-h1{
-    color:#6b37c1;
+
+/* Create and style grid container for the banner */
+
+.center {
+    margin-left: auto;
+    margin-right: auto;
+}
+
+/* .width-sm {
+    max-width: 20rem;
+} */
+
+.width-lg {
+    max-width: 40rem;
+}
+.width-xl {
+    max-width: 60rem;
+}
+
+#landing {
+    background-color: var(--primary-color);
+}
+
+.grid-container {
+    /* border: solid black 2px; */
+    display: grid;
+    grid-template-columns: repeat(3, minmax(10rem, 1fr));
+    grid-template-rows: repeat(7, 5rem);
+    grid-template-areas: 
```

Comment:
I think this is a great feature by defining the grid areas and their names, very explanatory because it explains the cells in the grid and by giving them the actual names for each section for the sites navigation 

Great work ✨